op-dispute-mon/mon/types/types.go

```diff
@@ -1,11 +1,17 @@
 package types
 
 import (
+	"math/big"
+
 	faultTypes "github.com/ethereum-optimism/optimism/op-challenger/game/fault/types"
 	"github.com/ethereum-optimism/optimism/op-challenger/game/types"
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// MaxUint128 is the maximum value of a uint128. This is used to compare against the bond amount.
+// If the bond amount is max uint128, the claim is resolved and the bond has already been claimed.
+var MaxUint128 = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 128), big.NewInt(1))
```

Comment:
Can we name this something more meaningful like `NoBond` or `ClaimResolvedBond` etc?  It's a symbolic value so we want the name to tell us what it's symbolic of not just what the number is.

Reply:
Yes totally, I think `NoBond` is fairly confusing since there most definitely was a bond. I'll make it `ResolvedBondAmount` for now pending your approval :)

Reply:
Yep, great name. :)